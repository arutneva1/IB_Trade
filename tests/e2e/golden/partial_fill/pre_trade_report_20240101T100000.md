NetLiq: 1000.00
Cash USD: 0.00
Cash Buffer: 0.00

| symbol | target_pct | current_pct | drift_bps | price | dollar_delta | share_delta | side | est_notional | reason |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AAA | 0.00 | 100.00 | -10000.00 | 100.00 | -1000.00 | -10.0000 | SELL | -1000.00 |  |
| TOTAL | 0.00 | 100.00 | -10000.00 |  | -1000.00 |  |  | -1000.00 |  |
