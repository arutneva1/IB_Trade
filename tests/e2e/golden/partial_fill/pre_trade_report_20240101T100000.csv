NetLiq,1000.00
Cash USD,0.00
Cash Buffer,0.00

symbol,target_pct,current_pct,drift_bps,price,dollar_delta,share_delta,side,est_notional,reason
AAA,0.0,100.0,-10000.0,100.0,-1000.0,-10.0,SELL,-1000.0,
TOTAL,0.0,100.0,-10000.0,,-1000.0,,,-1000.0,
